guides/common/modules/con_limiting-resources.adoc

```diff
@@ -0,0 +1,18 @@
+[id="limiting-resources"]
+= Limiting resources
+
+You can use the Resource Quota plugin to limit users' access to resources.
+
+Supported resource types::
+* CPU {endash} limit the number of CPU cores
+* Memory {endash} limit the amnount of memory
+* Disk {endash} limit the amount of disk space
+
+Supported compute resources::
+* VMware
+* libvirt
+
+Configuration options::
```

Comment:
Should I move this towards a "Configuring resource quotas plugin" module?

Reply:
That depends.
Are these configuration options determined as arguments to the `foreman-installer`? Or are they global settings?

Reply:
I would prefer after the installation step. The further text is talking about the settings which have not been introduced before and it feels a bit confusing to me. You can also mention the default values of the settings, if that is something which is commonly done in docs.